src/views/Admin/ReportingTool.vue

```diff
@@ -475,4 +562,9 @@ button {
   width: 100%;
   border-top: 2px solid rgba(75, 85, 99, 0.54);
 }
+.product-pie-card {
```

Comment:
Use tailwind class `flex items-center justify-center`

Reply:
Tailwind is not been used in this project